watertap/core/tests/test_zero_order_electricity.py

```diff
@@ -116,31 +115,7 @@ def test_solution(self, model):
 
     @pytest.mark.component
     def test_report(self, model):
-        stream = StringIO()
-
-        model.fs.unit.report(ostream=stream)
-
-        output = """
-====================================================================================
-Unit : fs.unit                                                             Time: 0.0
-------------------------------------------------------------------------------------
-    Unit Performance
-
-    Variables: 
-
-    Key                   : Value  : Fixed : Bounds
-       Electricity Demand : 420.00 : False : (0, None)
-    Electricity Intensity : 10.000 :  True : (None, None)
-
-------------------------------------------------------------------------------------
-    Stream Table
-    Empty DataFrame
-    Columns: []
-    Index: []
-====================================================================================
-"""
-
-        assert output == stream.getvalue()
+        model.fs.unit.report()
 
```

Comment:
So, I guess my issue with this is that we are no longer checking the values of the solutions on output. So this test is effectively testing nothing now.

Reply:
Most of the time, no. For example, this test already has a `test_solution` method for the "Electricity Demand".

I'm adding explicit solution tests in a few more places that I missed where we don't check it otherwise.